angularjs-portal-home/src/main/webapp/js/override.js

```diff
@@ -28,11 +25,17 @@ define(['angular'], function(angular) {
                 'entry': 'entry/',
                 'entries': 'entries.json',
             },
-            'notificationsURL': '/web/staticFeeds/notifications.json',
+            'messagesURL': '/web/staticFeeds/messages.json',
             'groupURL': '/portal/api/groups',
             'kvURL': '/storage',
             'loginSilentURL': '/portal/Login?silent=true&profile=bucky',
           },
+          'MESSAGES': {
+            'announcementsEnabled': true,
+            'notificationsEnabled': true,
```

Comment:
Do we need these flags, or could the enablement of announcements and notifications be implied by whether there are announcement-flavored and notification-flavored items in the messages feed? If there's no announcements in the feed, then they're not enabled. If there's no notifications in the feed, then they're not enabled.